components/org.wso2.carbon.identity.api.user.recovery/org.wso2.carbon.identity.rest.api.user.recovery.v1/src/main/java/org/wso2/carbon/identity/rest/api/user/recovery/v1/impl/core/utils/RecoveryUtil.java

```diff
@@ -261,6 +270,54 @@ public static String buildUri(String tenantDomain, String endpoint, String baseU
         return tenantQualifiedRelativePath + endpoint;
     }
 
+    /**
+     * Builds URI prepending the user API context with the proxy context path to the endpoint.
+     * Ex: /t/<tenant-domain>/api/users/<endpoint>
+     *
+     * @param endpoint Relative endpoint path.
+     * @return Relative URI.
+     */
+    public static String buildURIForBody(String tenantDomain, String endpoint, String baseUrl) {
+
+        String url;
+        String context = getContext(tenantDomain, endpoint, baseUrl);
+
+        try {
+            url = ServiceURLBuilder.create().addPath(context).build().getRelativePublicURL();
+        } catch (URLBuilderException e) {
+            String errorDescription = "Server encountered an error while building URL for response body.";
+            org.wso2.carbon.identity.api.user.common.error.ErrorResponse errorResponse =
```

Comment:
Can't we import this package without adding the full path here?

Reply:
Another package called ErrorResponse is also used in this class. Please refer https://github.com/wso2/identity-api-user/blob/9b07a372a1cd3b9ce4458a46573da9c643969c08/components/org.wso2.carbon.identity.api.user.recovery/org.wso2.carbon.identity.rest.api.user.recovery.v1/src/main/java/org/wso2/carbon/identity/rest/api/user/recovery/v1/impl/core/utils/RecoveryUtil.java#L39